apps/webClient/src/views/Home/index.tsx

```diff
@@ -74,14 +79,20 @@ const Home: React.FC<Props> = ({}) => {
               </Box>
             </Grid>
           </Grid>
-          <Grid item container sm={4} justify="flex-end" alignItems="flex-end">
-            <NavButton route="about" text="about us" isVertical={true} />
+          <Grid
+            item
+            container
+            sm={2}
+            justify="flex-end"
+            alignItems="flex-start"
+          >
+            <NavButton route="/about" text="about us" isVertical={true} />
           </Grid>
         </Grid>
       </Box>
       <Box mt={12}>
         <Grid container justify="center">
-          <NavButton route="lets-talk" text="let's talk" />
+          <NavButton route="/lets-talk" text="let's talk" />
```

Comment:
Easier to compare to location.pathname which has this char.

Reply:
works for me